lightsaber.ino

```diff
@@ -451,6 +451,7 @@ EFFECT(blaster);
 EFFECT(lockup);
 EFFECT(poweronf);
 EFFECT(font);   // also polyphonic
+EFFECT(bgnlock); // monophonic begin lock..?
```

Comment:
I think it's fine to use the same effect name for monophonic and polyphonic, since we can still tell which is is by looking at lockup.files_found()

Not a big deal, but it saves a few bytes of ram...

Reply:
I could. But doesn't that also make the number standard messed up? Monophonic files use a single digit suffix so you can get away with "endlock" (7 chars plus single digit. But polyphonic use two digit numbers so I needed a six char label, hence the bgnlck and endlck.

That was my rationale for making them different names.

Reply:
That's not how the effect class works in proffieOS. The numbering scheme is determined dynamically for each effect. So if you have a monophonic effect called "stab", you can use 0-4 digits, and it will all work as long as your naming is consistent.

I suggest that we always us bgnlock, both for polyphonic and monophonic fonts. For "endlock/endlck", we might want to consider having two different names to allow for a larger number of effects.

Of course, another option would be to implement a new (optional) naming scheme:

     EFFECTNAME/NNNNN.wav

By doing that, the effect could be up to 8 characters, and we could still have 100 million effects of each type. :)

Reply:
Yeah I actually wish fonts used folders for the name given we are getting up there... some fonts have 20 different blasts and 20 different clashes. Irrespective of naming constraints that gets to be just an unwieldy directory to look at. It triggers my OCD lol.

For now I will just rename bgnlck to bgnlock and leave the names different for endlock/endlck. But yeah I had done the polyphonic as lck suffix to preserve the ability for 0-2 digits. Vs 0-1 digits